PSModuleDevelopment/functions/utility/Show-PSMDSyntax.ps1

```diff
@@ -0,0 +1,206 @@
+﻿function Show-PSMDSyntax {
+    <#
+    .SYNOPSIS
+        Validate or show parameter set details with colored output
+
+    .DESCRIPTION
+        Analyze a function and it's parameters
+
+        The cmdlet / function is capable of validating a string input with function name and parameters
+
+    .PARAMETER CommandText
+        The string that you want to analyze
+
+        If there is parameter value present, you have to use the opposite quote strategy to encapsulate the string correctly
+
+        E.g. for double quotes
+        -CommandText 'Import-D365Bacpac -ImportModeTier2 -SqlUser "sqladmin" -SqlPwd "XyzXyz" -BacpacFile2 "C:\temp\uat.bacpac"'
+        
+        E.g. for single quotes
+        -CommandText "Import-D365Bacpac -ExportModeTier2 -SqlUser 'sqladmin' -SqlPwd 'XyzXyz' -BacpacFile2 'C:\temp\uat.bacpac'"
+
+    .PARAMETER Mode
+        The operation mode of the cmdlet / function
+
+        Valid options are:
+        - Validate
+        - ShowParameters
+
+    .PARAMETER IncludeHelp
+        Switch to instruct the cmdlet / function to output a simple guide with the colors in it
```

Comment:
I think this parameter should be named `-Legend` instead, as that's what it really enables

Also the description is slightly confusing:
- The switch part is redundant (the fact that it is a switch is shown as the type behind the name when calling help)
- cmdlet / function differentiation is not a subject needed
- Referring to the command you are explaining is counter-intuitive (we already _are_ explaining this command, so what is it referring to?)

Instead something like:
```
Include a legend explaining the color mapping / meaning
```
Would be very clear and concise about its purpose